src/client/client.ts

```diff
@@ -263,7 +263,15 @@ export function createDfuseClient(options: DfuseClientOptions): DfuseClient {
   const secureEndpoint = options.secure === undefined ? true : options.secure
   const authentication = options.authentication === undefined ? true : options.authentication
 
-  const authUrl = options.authUrl || (authentication ? "https://auth.dfuse.io" : "null://")
+  let defaultAuthUrl = "null://"
+
+  if (authentication && isEnterpriseEdition(endpoint)) {
+    defaultAuthUrl = "https://auth.dfuse.io"
+  } else if (authentication && isCommunityEdition(endpoint)) {
```

Comment:
I will refactor to only consider Nation if it matches `isCommunityEdition` and always fallback to standard dfuse auth otherwise (so I'll remove isEnterpriseEdition condition).